backend/onedrive/onedrive_test.go

```diff
@@ -19,6 +19,17 @@ func TestIntegration(t *testing.T) {
 	})
 }
 
+// TestIntegrationCn runs integration tests against the remote
+func TestIntegrationCn(t *testing.T) {
+	fstests.Run(t, &fstests.Opt{
+		RemoteName: "TestOneDriveCn:",
+		NilObject:  (*Object)(nil),
```

Comment:
Additional integration tests that require specific remote should be skipped if `test_all` overrides their remote.
See example in [webdav_test.go](https://github.com/rclone/rclone/blob/v1.53.4/backend/webdav/webdav_test.go#L20)

Reply:
You can just cherry-pick https://github.com/ivandeex/rclone/commit/32c682e1194b3d176e94f7d9d4f8bb58e4f5d32b

Reply:
Thanks, seems that you've pushed these changes.
Have you received my config?

Reply:
Yes, received. I ran integration test using a combination of your settings for TestOnedriveCN and and my Onedrive global/personal for TestOnedrive.

Reply:
AFAIK we can't.
I think you are worried by `rclone-test-xxx` piling after tests (rclone creates a random directory for each subtest and removes it after success but leaves for investigations if the test failed). I just use the following one-liner periodically to purge them
```
rclone lsd TestRemote: | awk '/rclone-test-/{print$5}' |xargs -tri rclone purge TestRemote:{}
```

Reply:
Yep that's what I'm worrying :( Thanks a lot for your oneliner, I'll try it next time.
But can we use the alias backend to achieve it?

Reply:
Many tests cases depend on properties of remote under test.
I don't recommend using alias. It can confuse them.